[DM-1294]: Added 3 announcements for device grid changes. (#241)
SoftwareAG/c8y-releasenotes: content/release-2024-0/announcements-2024-0.md

#### Implemented

##### Removal of deprecated device-grid model classes, column implementations and services

> As announced with [release 10.17](/release-10-17-0/announcements-10-17-0) shared classes, components and services from the device-grid package are deprecated and new ones are created in the core data-grid component. All those deprecated classes are now removed.

Reviewer: ```suggestion
As announced earlier, most recently with [release 10.17](/release-10-17-0/announcements-10-17-0), shared classes, components and services from the device-grid package are deprecated and have now been removed. New ones are created in the core data-grid component.
```
